renderer.js

```diff
@@ -0,0 +1,48 @@
+function getMapping() {
+    return [{id: "awsAccessKeyId", key: "AWS_ACCESS_KEY_ID"},
+            {id: "awsSecretAccessKey", key: "AWS_SECRET_ACCESS_KEY"},
+            {id: "endpoint", key: "ENDPOINT"},
+            {id: "osdPath", key: "osdPath"}
```

Comment:
Do we need extra logic to validate that the path provided is actually an OSD path? What if user put some other random links or other aws resources links

Reply:
I will look into adding validation. I think the simplest would be to try to run the curl command and just output the stderr.